.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
   directories:
   - _build
   - deps
-  - smart-home-auth-server/deps
+  - smart-home-auth-server-deps
```

Comment:
is this just so you can cache the deps? 💭

Reply:
Yeah, couldn't get it to work however and have commented it out in 9202264

Reply:
Fair enough. A worthwhile endeavour if possible, but not a deal-breaker if not. 👍